src/main/java/org/broadinstitute/hellbender/engine/spark/SparkCommandLineArgumentCollection.java

```diff
@@ -45,4 +59,24 @@ public String getSparkMaster() {
         return sparkMaster;
     }
 
+    public String getSparkVerbosity(final Log.LogLevel toolVerbosity) {
```

Comment:
Can you add a javadoc summarizing this behavior?

Reply:
Done